loadtest/control/simplecontroller/actions.go

```diff
@@ -100,6 +100,55 @@ func (c *SimpleController) createPost() control.UserStatus {
 	return c.newInfoStatus(fmt.Sprintf("post created, id %v", postId))
 }
 
+func (c *SimpleController) addReaction() control.UserStatus {
+	// get posts from UserStore that have been created in the last minute
+	posts, err := c.user.Store().PostsSince(time.Now().Unix()*1000 + 60000)
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	if len(posts) == 0 {
+		return c.newInfoStatus("no posts to add reaction to")
+	}
+
+	err = c.user.SaveReaction(&model.Reaction{
+		UserId:    c.user.Store().Id(),
+		PostId:    posts[0].Id,
+		EmojiName: "grinning",
```

Comment:
we should probably randomize this?

Reply:
This is meant to be this way. If you notice we don't use any random function in any of the SimpleController's actions. This is because I want to keep it as deterministic as possible.

We'll get to a more realistic controller once we know more about how the system behaves as a whole. There's a tuning process needed to get to that point so for now my goal is to keep the number of variables down to a minimum, hence the lack of randomization.